Source/WebKit/UIProcess/ApplicationStateTracker.mm

```diff
@@ -50,6 +51,28 @@ - (BOOL)_isHostedInAnotherProcess;
 
 namespace WebKit {
 
+static WeakHashSet<ApplicationStateTracker>& allApplicationStateTrackers()
+{
+    static NeverDestroyed<WeakHashSet<ApplicationStateTracker>> trackers;
+    return trackers;
+}
+
+static void updateApplicationBackgroundState()
+{
+    static bool isApplicationInBackground = false;
+    auto isAnyStateTrackersInForeground = []() -> bool {
+        return WTF::anyOf(allApplicationStateTrackers(), [](auto& tracker) {
+            return !tracker.isInBackground();
+        });
+    };
+    bool currentIsApplicationInBackground = !isAnyStateTrackersInForeground();
```

Comment:
"current" can be ambiguous at a time of change like this.

If you switch to "s_" then you can just call this isApplicationInBackground.